### DISCUSSION TITLE
Migrate existing project from V8 to V9 doesn't work as it should

### DISCUSSION THREAD

**Discussion by ghost** [webdriverio/webdriverio#13475] [Technical Discussion]
Hello,
I upgraded my project to V9, but it seems that it doesn't work as it should.
Version is 9 in package.json
But, for example, the isDisplayedInViewport() function doesn't appears as deprecated and isDisplayed() still doesn't accept param.
Maybe I missed a step ?
Thanks for your help
<img width="713" alt="Capture d’écran 2024-08-29 à 11 01 39" src="https://github.com/user-attachments/assets/5e21c1f2-0c7b-478d-b3c3-a28e9ace37f4">
<img width="278" alt="Capture d’écran 2024-08-29 à 11 01 29" src="https://github.com/user-attachments/assets/d327676b-dcfc-4b05-9e22-6c319552c2e0">
<img width="305" alt="Capture d’écran 2024-08-29 à 11 01 03" src="https://github.com/user-attachments/assets/0473f816-bbce-4f1f-9c8a-cdd4bfba1df3">
<img width="316" alt="Capture d’écran 2024-08-29 à 11 00 51" src="https://github.com/user-attachments/assets/822f92ea-c79e-457a-9d16-9850e3b9b4f7">

**Comment by christian-bromann** [comment]
@ichman please make sure to remove your `package-lock.json` and install all WDIO related packages to `latest`. If you still encounter the issue please provide a reproducible example and I am happy to take a look at it.

**Reply by ghost** [reply]
![Uploading Capture d’écran 2024-08-30 à 10.05.54.png…]()
Hello @christian-bromann thanks for your answer.
I did what you explained but still the same here.
I deleted the package-lock.jon then installed all related packages to latest (cf screenshots)
<img width="447" alt="Capture d’écran 2024-08-30 à 10 01 00" src="https://github.com/user-attachments/assets/fc27b421-4172-4264-afa2-0ef6eb474b10">
<img width="803" alt="Capture d’écran 2024-08-30 à 10 01 11" src="https://github.com/user-attachments/assets/f3a57093-972c-44f9-b173-4d25286c0df2">
It's not easy to provide an example because it's a huge project :) 
Maybe I could try to set up a new project in V8 then migrate it to V9 and see what happens

**Reply by ghost** [reply]
on this repo -> https://github.com/ichman/migrate (command npx wdio run ./wdio.conf.js / I add isDisplayed / isDisplayedInViewport)
it works fine, so I guess I have to find what's blocking

**Comment by ghost** [comment]
Hello, deleting following packages in the package.json before "npm install" and upgrade solved the issue.
```
        "wdio-chromedriver-service": "^8.1.1",
        "wdio-intercept-service": "^4.4.1",
        "wdio-wait-for": "^3.0.11"
```
It works fine now 👍 
Thanks
